addons/point_of_sale/models/pos_config.py

```diff
@@ -233,7 +233,8 @@ def _compute_allowed_pricelist_ids(self):
     @api.depends('company_id')
     def _compute_company_has_template(self):
         for config in self:
-            if config.company_id.chart_template_id:
+            has_accounting_entries = self.env['account.chart.template'].existing_accounting(config.company_id)
+            if config.company_id.chart_template_id or has_accounting_entries:
                 config.company_has_template = True
             else:
                 config.company_has_template = False
```

Comment:
```suggestion
            config.company_has_template = self.env['account.chart.template'].existing_accounting(config.company_id)
```

Reply:
```suggestion
            config.company_has_template = self.env['account.chart.template'].existing_accounting(config.company_id) or config.company_id.chart_template_id
```

Would we need to check whether the chart_template_id exists in the case where there's no accounting entry but they do have a chart template set?

Reply:
Indeed because the user is free to import a COA from an external source like from a csv containing the accounts

Reply:
Hello @smetl ,

I committed the suggested change!